covalent_ssh_plugin/ssh.py

```diff
@@ -227,7 +234,7 @@ def execute(
                 app_log.debug(f"exception: {exception}")
                 raise exception
 
-        self.client.close()
+        conn.close()
```

Comment:
just a thought... and I think the functionality for this currently does not exists... but to promote re-use of existing SSH connections...  run conn.close only by the last electron that is using this executor?

Reply:
I think we can do that, but that might be out of the scope of this PR.